docs/source/pythonapi/examples/mgxs-part-iv.ipynb

```diff
@@ -513,7 +513,7 @@
    "source": [
     "# Specify multi-group cross section types to compute\n",
     "mgxs_lib.mgxs_types = ['total', 'absorption', 'nu-fission', 'fission',\n",
-    "                       'nu-scatter matrix', 'scatter matrix', 'chi']"
+    "                       'nu-scatter matrix', 'multiplicity matrix', 'chi']"
```

Comment:
I haven't done one of my good old convergence tests on it, but it looks like (and stands to reason) the nu-fission matrix is slower to resolve than good ol' nu-fission and chi.  When I used nu-fission matrix the CE-MG bias was 1,000pcm at these few history levels, dropping to low levels (<20pcm i think it was) as histories increased.  Since I don't want to artificially scare anyone away from using MG mode with a high bias, I stuck with the nu-fission/chi pair.